pipeline/scripts/ci/upstream_cli.py

```diff
@@ -0,0 +1,166 @@
+"""
+This script provides the upstream latest rpm repo path and container image.
+Please note that this script should be run under root privilages
+inorder to pull the image and podman should be installed on that host.
+"""
+import fcntl
+import logging
+import os
+import re
+import sys
+import time
+
+import requests
+import yaml
+from docopt import docopt
+from packaging import version
+
+LOG = logging.getLogger(__name__)
+
+
+def store_in_upstream(branch_name, repo, image, upstream_version):
+    """Method to update it in upstream.yaml.
+    Args:
+        branch_name: Name of upstream branch
+        repo: upstream branch compose URL
+        image: upstream branch image
+        version: upstream branch ceph version.
+    """
+    file_name = "/ceph/cephci-jenkins/latest-rhceph-container-info/upstream.yaml"
+    stream = open(file_name, "r")
+    repo_details = yaml.load(stream)
+    # To handle new release which is not updated yet
+    if repo_details[branch_name]:
```

Comment:
would it make sense to update the release.yaml if release entry is not found? instead of raising exception.

This would give us a way to update dynamically for newer releases.

Reply:
Yes @sunilkumarn417 it makes sense to update dynamically Instead of raising exceptions I updated the file,
Please review it.